CONTRIBUTING.md

```diff
@@ -5,29 +5,37 @@
 
 ## Getting Started
 
-To work on this project please create an account on Cloudflare and Discord.
-Also, create a personal discord server.
-Copy the guild id of your server and save it in a .env file as `DISCORD_GUILD_ID`
-
-To get the server ID for the first parameter, open Discord, go to Settings > Advanced and enable developer mode.
-Then, right-click on the server title and select "Copy ID" to get the guild ID.
-
+To contribute to this project, you have to set up few things first.
+
+- Create an account on Cloudfare Workers and Discord if you don't already have one
+- Create a personal discord server
+- Fork and clone this repo
+- Install volta - this will check if you have the right node version or not
+- Create a .env file
```

Comment:
mention that to be created in root directory

Reply:
done